app/models/scc_account.rb

```diff
@@ -268,7 +268,7 @@ def revalidate_subscription_status(elements, scc_id)
   # params: items_to_invalidate: ActiveRecord_(*)
   #         save_record: store in database or not (default)
   # return: ActiveRecord elements with invalidated subscription status
-  def invalidate_subscription_status(items_to_invalidate, save_record = false)
+  def invalidate_subscription_status(items_to_invalidate, save_record:)
```

Comment:
Is this really the correct way of specifying that the default-value will be `false`?

Reply:
This is not specifying the default value anymore, it is defining `save_record` as keyword argument. Means that you need to pass it via keyword explicitly.